web_generator/test/integration/interop_gen/project/input/e.d.ts

```diff
@@ -0,0 +1,27 @@
+export declare class Configuration {
```

Comment:
I'm guessing the purpose of this file is just so it can be correctly ignored?

What happens if a file imports this file and exports some of its members but it's not in the config? Is there an error in the generator?

Reply:
Ooh interesting. That sounds good to me then. An error would prevent confusion but I think it's fine to somewhat handle this.